homeassistant/components/github/coordinator.py

```diff
@@ -136,3 +139,26 @@ async def _async_update_data(self) -> GitHubResponseModel[dict[str, Any]]:
         else:
             self._last_response = response
             return response.data["data"]["repository"]
+
+    async def _handle_event(self, event: GitHubEventModel) -> None:
+        """Handle an event."""
+        if event.type in REFRESH_EVENT_TYPES:
+            await self.async_request_refresh()
+
+    @staticmethod
+    async def _handle_error(error: GitHubException) -> None:
+        """Handle an error."""
+        LOGGER.error("An error occurred while processing new event - %s", error)
```

Comment:
Is the event name included in the error?

Reply:
no, this happens when there is an issue fetching all new events, before single events are constructed.
changed "new event" -> "new events"